ocaml/typing/solver.ml

```diff
@@ -242,7 +244,8 @@ module Solver_mono (C : Lattices_mono) = struct
   let apply :
       type a b l r.
       b C.obj -> (a, b, l * r) C.morph -> (a, l * r) mode -> (b, l * r) mode =
-   fun dst morph -> function
+   fun dst morph m ->
```

Comment:
Does this change make a difference? I don't think it should, but would be very interested if it does!

Reply:
My impression is that with Nick's n-ary function patch, it might?

Reply:
A reasonable guess, but I don't think so: there's a special case in that design just for this pattern, of `fun ... -> function`, where the `function` is considered part of the `fun`, really. I don't care much style-wise one way or another; I was mostly asking to see if this was observed to make a difference, which would be a very interesting little result.

Reply:
Yeah I just read the blog post and you are right. This was done together with other changes, so it's hard to say if it has effects at all.